### PR TITLE
Fixed further issues when deploying in dev mode

### PR DESCRIPTION
**Pull request by daurer**

### CODE REVIEW

File: charts/argocd-apps/values.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ destination:
   server: https://kubernetes.default.svc
 
 events:
-  enabled: false
+  enabled: true
```
Comment (garryod) [inline charts/argocd-apps/values.yaml:5]: Is this production ready now?

Comment (daurer) [reply charts/argocd-apps/values.yaml:5]: Good catch, I'd say probably not...

**Review by garryod — DISMISSED** [review]
LGTM

**Review by garryod — APPROVED** [review]
LGTM

**Pull request merged**